docs/excel/excel-add-ins-troubleshooting.md

```diff
@@ -41,6 +41,56 @@ The following APIs are affected by this workbook switch:
 
 See [Coauthoring in Excel add-ins](co-authoring-in-excel-add-ins.md) for patterns to use with events in a coauthoring environment. The article also discusses potential merge conflicts when using certain APIs, such as [`TableRowCollection.add`](/javascript/api/excel/excel.tablerowcollection#add-index--values-).
 
+## Known Issues
+
+### Binding events ID discrepancy
+
+Both [BindingDataChangedEventArgs.binding](/javascript/api/excel/excel.bindingdatachangedeventargs#binding) and [BindingSelectionChangedEventArgs.binding](/javascript/api/excel/excel.bindingselectionchangedeventargs#binding) return a temporary `Binding` object that contains the ID of the `Binding` object that raised the event. Use this ID with `BindingCollection.getItem(id)` to retrieve the `Binding` object that raised the event.
+
+The following code sample first shows how to assign an event listener to a binding, and then call the `getBindingId` method when the `onDataChanged` event is triggered. In the `getBindingId` method, the code sample then demonstrates how to use the ID of the temporary `Binding` object to retrieve the `Binding` object that raised the event.
+
+```js
+Excel.run(function (context) {
+    // Retrieve your binding.
+    var binding = context.workbook.bindings.getItemAt(0);
+
+    return context.sync().then(function () {
+        // Register an event listener to detect changes to your binding
+        // and then trigger the `getBindingId` method on change. 
+        binding.onDataChanged.add(getBindingId);
+
+        return context.sync();
+    });
+});
+
+function getBindingId(eventArgs) {
+    return Excel.run(function (context) {
+        // Get the temporary binding object and load its ID. 
+        var tempBindingObject = eventArgs.binding;
+        tempBindingObject.load("id");
+
+        // Use the temporary binding object's ID to retrieve the original binding object. 
+        var originalBindingObject = context.workbook.bindings.getItem(tempBindingObject.id);
+
+        // You now have the binding object that raised the event: `originalBindingObject`. 
+    });
+}
+```
+
+### Cell format `useStandardHeight` and `useStandardWidth` issues
+
+The [useStandardHeight](/javascript/api/excel/excel.cellpropertiesformat#useStandardHeight) property of `CellPropertiesFormat` doesn't work properly in Excel on the web. Due to a bug in the Excel on the web UI, setting the `useStandardHeight` property to `true` will calculate height imprecisely on this platform. For example, a standard height of **14** will be modified to **14.25** in Excel on the web.
```

Comment:
Is there a GitHub issue in office-js tracking this issue? We could link to it from here.

Reply:
According to [internal tracking](https://office.visualstudio.com/OC/_workitems/edit/4588134), this is not an API issue, it's a platform issue related to pixel sizing conversion. I don't think there's anything public I can link to.